shared/src/test/kotlin/com/egm/stellio/shared/model/ExpandedEntityTests.kt

```diff
@@ -0,0 +1,82 @@
+package com.egm.stellio.shared.model
+
+import com.egm.stellio.shared.util.NgsiLdParsingUtils
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class ExpandedEntityTests {
+
+    @Test
+    fun `it should get relationships from ExpandedEntity`() {
+        val expandedEntity = NgsiLdParsingUtils.parseEntity(
+            "{\n" +
```

Comment:
Feel free to use multiline strings wherever you can :)

Reply:
Thanks for the tip, it is much more easier to use indeed